### PR TITLE
[develop] Trello-378g7zcn: Prettify currencies page

### PR DESCRIPTION
**Pull request by wildeastengineer**
[Trello-378g7zcn](https://trello.com/c/378g7zcn): Prettify currencies page

---

- [x] Markup and styles
- [x] Translations
- [x] API calling

### CODE REVIEW

File: client/components/common/RadioButton/radioButton.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    align-items: center;
+    display: flex;
+    justify-content: center;
+    height: 100%;
+    width: 100%;
+
+    input {
+        margin: 0;
+    }
+}
```
Comment (AndreySurzhan) [inline client/components/common/RadioButton/radioButton.scss]: additional line is needed

File: client/components/pages/ProjectSettings/Currencies/CurrenciesList.jsx [hunk 2]
```diff
@@ -1,35 +1,48 @@
...
 const defaultProps = {
     currencies: [],
-    actionTitle: '',
-    onButtonClick: () => {}
+    isUsed: false,
+    isDefault: false
 };
 
-function CurrenciesList({ currencies, actionTitle, onButtonClick }) {
+function CurrenciesList({
```
Comment (AndreySurzhan) [inline client/components/pages/ProjectSettings/Currencies/CurrenciesList.jsx]: strange formatting

File: client/store/currencies/selectors.js [hunk 3]
```diff
@@ -1,6 +1,13 @@
...
-export const getGlobalCurrenciesList = (state) => (getGlobalCurrencies(state).data);
-export const getProjectCurrenciesList = (state) => (getProjectCurrencies(state).data);
+export const getGlobalCurrenciesList = (state) => (getGlobalCurrenciesState(state).data);
+export const getProjectCurrenciesList = (state) => (getProjectCurrenciesState(state).data);
+export const getMainCurrency = (state) => (getMainCurrencyState(state).data);
+export const getMainCurrencyId = (state) => {
+    const mainCurrency = getMainCurrency(state);
+
+    return mainCurrency ? mainCurrency._id : null;
+};
```
Comment (AndreySurzhan) [inline client/store/currencies/selectors.js]: line

File: client/components/pages/ProjectSettings/Currencies/CurrenciesList.jsx (hunk 2 above)
Comment (wildeastengineer) [inline client/components/pages/ProjectSettings/Currencies/CurrenciesList.jsx]: I think it's ok for long params list

**Comment by wildeastengineer** [comment]
@AndreySurzhan thanks for review

**Pull request merged**